src/test/java/BringItOn/page/AbstractPage.java

```diff
@@ -0,0 +1,22 @@
+package BringItOn.page;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public abstract class AbstractPage {
```

Comment:
In your project you have 2 AbstractPage classes and if your project is extended further it may cause confusion. The best practice here would be to have one Abstract page for all your tests, but if to leave this implementation at least the naming of the Abstract class could be specified - e.g. AbstractBringItOn class.

Reply:
Appreciated for your feedback! I was trying to keep those two tasks separated, but still wanted them to share the same project, so I have it divided into one level packages. You're right, doing it this way, I might have named them differently.